covid_helper.py

```diff
@@ -9,6 +9,51 @@
 
 base_color = sns.color_palette()[0]
 
+def save_GUS_data(path):
+    gus_path = path + '/' + 'gus_data'
+
+    # create a subfolder for geo data
+    if not os.path.exists(gus_path):
+        os.makedirs(gus_path)
+
+    # Chosen Variables:
+    # 72305 - liczba mieszkancow
+    # 746289 - mediana wieku ludnosci
+    # 1601437 - liczba boisk futbolowych - tylko 2018
+    # 1241 - liczba muzeow
+    # 64428 - przecietne wynagrodzenie brutto
+
+    variables = {
+        72305: ['liczba mieszkancow', 2019, 'Mieszkancy'],
+        746289: ['mediana wieku ludnosci',2019, 'Mediana'],
+        1601437: ['liczba boisk futbolowych', 2018, 'Boiska'],
+        1241: ['liczba muzeow', 2019, 'Muzea'],
+        64428: ['przecietne wynagrodzenie brutto', 2019, 'Wynagrodzenie']
+    }

```

Comment:
DaneGUS(72305, 'liczba mieszkancow', 2019, 'Mieszkancy')
DaneGUS.id -> 72305

for gus in dane_gus:
    url = f'https://bdl.stat.gov.pl/api/v1/data/by-variable/{gus.id}?unit-level={unit_level}&year={gus.rok}'